### ISSUE TITLE
Softmax Regression tutorial markup error

### ISSUE THREAD

**Issue by zseder** [lisa-lab/pylearn2#1155]
At line https://github.com/lisa-lab/pylearn2/blob/master/pylearn2/scripts/tutorials/softmax_regression/softmax_regression.ipynb#L295, there is `We use \"*train\" to define the training set. The * is YAML`, but with the asterisk, without being escaped.

**Comment by lamblin** [comment]
Fixed in #1418.
